src/config/express.ts

```diff
@@ -12,6 +14,70 @@ const createServer = (): express.Application => {
     res.send('UP');
   });
 
+  app.post('/traders', (_req: any, res) => {
+    const { name } = _req.body;
+
+    try {
+      const response = stockMarket.createTrader(name);
+
+      res.send(response);
+    } catch (err: any) {
+      logger.error(err);
+      res.send(err.message);
+    }
+  });
+
+  app.patch('/traders/:trader_id/trades/buy', async (_req: any, res) => {
+    const { params: { trader_id }, body: { symbol, date, quantity } } = _req;
+
+    try {
+      await stockMarket.buy(trader_id, symbol, new Date(date), quantity);
+
+      res.send({
+        status: 'success'
+      });
+    } catch (err: any) {
+      logger.error(err);
+      res.send({
+        status: 'fail',
+        error: err.message
+      });
```

Comment:
`res.status(...).json(...) ` might fit this response format better.  `.json` generally when sending json, and status code to give more context to any error message.

 EG 400 bad request, 404 trader not found, 500 something went wrong, etc.